crane/config.go

```diff
@@ -229,6 +268,25 @@ func (c *config) initialize() {
 		}
 		c.containerMap[name] = container
 	}
+
+	c.setNetworkMap()
+	c.setVolumeMap()
+}
+
+func (c *config) setNetworkMap() {
+	c.networkMap = make(map[string]Network)
+	for rawName, network := range c.RawNetworks {
+		network.RawName = rawName
```

Comment:
I guess better nil handling here would allow
```
networks:
  foo:
```

to be valid without the `{}` to force the allocation of an object when unmarshalling